src/lib/s3.js

```diff
@@ -0,0 +1,55 @@
+const { ClearResult } = require("./utils");
```

Comment:
mm.. should have a trailing line break, eslint should have picked this one up, maybe we're missing a config?